examples/with-actix/src/main.rs

```diff
@@ -0,0 +1,101 @@
+use actix_web::{App, HttpServer, get, post, HttpResponse, Responder, web};
+use serde::Deserialize;
+
+mod prisma;
+use prisma::{user, post};
+
+extern crate dotenv;
+
+use dotenv::dotenv;
+
+#[get("/users")]
+async fn get_users() -> impl Responder {
+    let client = prisma::new_client().await.unwrap();
```

Comment:
A better patten than creating a new client for each request would be to create one client, attach it to the Actix App as state and use that within each service.